grazer/src/main/webapp/WEB-INF/views/personGraph.jsp

```diff
@@ -26,7 +26,7 @@
         <a href="<c:url value="/concept/${edge.targetNode.conceptId}" />">${edge.targetNode.label}</a>
     </c:when>
     <c:otherwise>
-        ${edge.targetNode.label}
+        <a href="<c:url value="/concept/${edge.targetNode.conceptId}" />">${edge.targetNode.label}</a>
     </c:otherwise>
```

Comment:
If <c:when test="${edge.targetNode.type == 'http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab' }" > and <c:otherwise> result in the same value : <a href="<c:url value="/concept/${edge.targetNode.conceptId}" />">${edge.targetNode.label}</a>, you can omit the test in c:when tag and use only c:otherwise to capture the result